pmlc/target/intel_gen/BUILD

```diff
@@ -35,6 +35,7 @@ plaidml_cc_library(
         "//pmlc/dialect/pxa/transforms",
         "//pmlc/dialect/tile/transforms",
         "//pmlc/dialect/stdx/transforms",
+        "//pmlc/rt/vulkan",
```

Comment:
The PRNG and linking issue are both fixed in Frank's PR https://github.com/plaidml/plaidml/pull/1329